sirepo/package_data/static/js/raydata.js

```diff
@@ -377,11 +380,14 @@ SIREPO.app.directive('scansTable', function() {
             $scope.images = null;
             $scope.noScansReturned = false;
             $scope.orderByColumn = 'start';
+            $scope.pdfSelectAllScans = false;
+            $scope.pdfSelectedScans = [];
```

Comment:
I think making this a dict instead of an array will make the operations on it easier.

Reply:
34d5c6d1b0aa5be7fb841e47a6fbf0fc34ecd9e9